### ISSUE TITLE
triple square brace (]) on if statement doesn't raise error

### ISSUE THREAD

**Issue by ciaranmcveigh5** [koalaman/shellcheck#1680]
#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
x=3

if [[ $x == 3 ]]]; then
echo hello
fi

```
Terminal Output

```
-bash: test4.sh: line 5: syntax error in conditional expression
-bash: test4.sh: line 5: syntax error near `;'
-bash: test4.sh: line 5: `if [[ $x == 3 ]]]; then'
```

Shellcheck Version

```
ShellCheck - shell script analysis tool
version: 0.7.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

#### Here's what shellcheck currently says:
Nothing - no errors output

#### Here's what I wanted or expected to see:
^-- SCXXXX: Error triple ]]] on if statement (or something similar)

**Comment by koalaman** [comment]
Indeed! This, along with `]]: then`, `]]#foo` and `]]then` was silently being ignored. You now get a warning. Thanks!
